lms/djangoapps/courseware/access.py

```diff
@@ -157,7 +157,7 @@ def has_access(user, action, obj, course_key=None):
         return _has_access_descriptor(user, action, obj, course_key)
 
     if isinstance(obj, CCXLocator):
-        return _has_access_ccx_key(user, action, obj)
+        return _has_access_course_key(user, action, obj)
```

Comment:
Can we remove this `if` statement entirely and let it fall through to the `if isinstance(obj, CourseKey):` below? CCXLocator is a CourseLocator which is a CourseKey, so I believe that should just work. I'd like to remove CCX special handling wherever possible.